types/devexpress-web/README.md

```diff
@@ -2,8 +2,8 @@ DevExpress ASP.NET/MVC TypeScript definitions
 =============================================
 
 You can use these TypeScript definitions for your web projects, which contain
-DevExpress ASP.NET or MVC controls. For this, simply add  at the top of your
-code `<reference path="TypeScript/devexpress-web.d.ts" />`.
+DevExpress ASP.NET or MVC controls. For this, simply add at the top of your
+code `<reference path="index.d.ts" />`.
```

Comment:
Hi @andy-ms. I've removed mention of the reference path from the README file. We plan to update the article about using TS definition in our documentation. With the new version of DevExpress controls (and their TS definition resp.) I'll add a link to this article.